### ISSUE TITLE
Activity stream fails if solr is down

### ISSUE THREAD

**Issue by alastair** [MTG/freesound#726]
```
Traceback:
File "/home/alastair/code/freesound/follow/views.py" in stream
  186.         'users_sounds': users_sounds,

Exception Type: UnboundLocalError at /home/stream/
Exception Value: local variable 'users_sounds' referenced before assignment
```

This happens because of the check at https://github.com/MTG/freesound/blob/master/follow/views.py#L173-L178
We should set a default for `users_sounds` and `tags_sounds`, but also improve the error checking to only catch the specific solr error exception.

**Comment by andrebola** [comment]
I think this issue is related with the closed #772

**Comment by ffont** [comment]
Yeah, so it seems. But the catch exception part can still be improved as alastair suggests.
